camel/bots/slack/slack_app.py

```diff
@@ -61,18 +61,21 @@ class SlackApp:
     def __init__(
         self,
         token: Optional[str] = None,
+        app_token: Optional[str] = None,
         scopes: Optional[str] = None,
         signing_secret: Optional[str] = None,
         client_id: Optional[str] = None,
         client_secret: Optional[str] = None,
         redirect_uri_path: str = "/slack/oauth_redirect",
         installation_store: Optional[AsyncInstallationStore] = None,
+        socket_mode: bool = True,
```

Comment:
Hi @Asher-hss , when we create a new slack app, socket mode is turned off by default. Let's set socket_mode to `False` to maintain consistency with the official configuration.

Reply:
sure